<ui version="4.0"> <class>SRUPathCalcForm</class> <widget class="QDialog" name="SRUPathCalcForm"> <property name="windowTitle"> <string>Расчёт маршрута SRU</string> </property> <layout class="QFormLayout"> <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Старт (Lat):</string></property></widget></item> <item row="0" column="1"><widget class="QDoubleSpinBox" name="spinStartLat"></widget></item> <item row="1" column="0"><widget class="QLabel"><property name="text"><string>Старт (Lon):</string></property></widget></item> <item row="1" column="1"><widget class="QDoubleSpinBox" name="spinStartLon"></widget></item> <item row="2" column="0"><widget class="QLabel"><property name="text"><string>Финиш (Lat):</string></property></widget></item> <item row="2" column="1"><widget class="QDoubleSpinBox" name="spinEndLat"></widget></item> <item row="3" column="0"><widget class="QLabel"><property name="text"><string>Финиш (Lon):</string></property></widget></item> <item row="3" column="1"><widget class="QDoubleSpinBox" name="spinEndLon"></widget></item> <item row="4" column="0"><widget class="QLabel"><property name="text"><string>Скорость (уз):</string></property></widget></item> <item row="4" column="1"><widget class="QDoubleSpinBox" name="spinSpeed"></widget></item> <item row="5" column="1"><widget class="QPushButton" name="buttonCalculate"><property name="text"><string>Рассчитать</string></property></widget></item> </layout> </widget> <resources> <connections> </connections></resources></ui>
